--- a/Assets/UI_fgx/HB_9sliceSolid.png.meta	2026-08-17 12:48:55.883913133 +0000
+++ b/Assets/UI_fgx/HB_9sliceSolid.png.meta	2026-08-17 12:48:55.915678352 +0000
@@ -49,7 +49,7 @@
   alignment: 0
   spritePivot: {x: 0.5, y: 0.5}
   spritePixelsToUnits: 100
-  spriteBorder: {x: 24, y: 24, z: 24, w: 24}
+  spriteBorder: {x: 6, y: 6, z: 6, w: 6}
   spriteGenerateFallbackPhysicsShape: 1
   alphaUsage: 1
   alphaIsTransparency: 1
--- a/Assets/UI_fgx/HB_9sliceSolidHov.png.meta	2026-08-17 12:48:55.890879573 +0000
+++ b/Assets/UI_fgx/HB_9sliceSolidHov.png.meta	2026-08-17 12:48:55.916416332 +0000
@@ -49,7 +49,7 @@
   alignment: 0
   spritePivot: {x: 0.5, y: 0.5}
   spritePixelsToUnits: 100
-  spriteBorder: {x: 24, y: 24, z: 24, w: 24}
+  spriteBorder: {x: 6, y: 6, z: 6, w: 6}
   spriteGenerateFallbackPhysicsShape: 1
   alphaUsage: 1
   alphaIsTransparency: 1
--- a/Assets/UI_fgx/HB_9sliceSolidClick.png.meta	2026-08-17 12:48:55.897275317 +0000
+++ b/Assets/UI_fgx/HB_9sliceSolidClick.png.meta	2026-08-17 12:48:55.916565765 +0000
@@ -49,7 +49,7 @@
   alignment: 0
   spritePivot: {x: 0.5, y: 0.5}
   spritePixelsToUnits: 100
-  spriteBorder: {x: 24, y: 24, z: 24, w: 24}
+  spriteBorder: {x: 6, y: 6, z: 6, w: 6}
   spriteGenerateFallbackPhysicsShape: 1
   alphaUsage: 1
   alphaIsTransparency: 1
--- a/Assets/UI_fgx/HB_9slice_FineEdge.png.meta	2026-08-17 12:48:55.903902672 +0000
+++ b/Assets/UI_fgx/HB_9slice_FineEdge.png.meta	2026-08-17 12:48:55.918730754 +0000
@@ -49,7 +49,7 @@
   alignment: 0
   spritePivot: {x: 0.5, y: 0.5}
   spritePixelsToUnits: 100
-  spriteBorder: {x: 10, y: 10, z: 10, w: 10}
+  spriteBorder: {x: 5, y: 5, z: 5, w: 5}
   spriteGenerateFallbackPhysicsShape: 1
   alphaUsage: 1
   alphaIsTransparency: 1
--- a/Assets/UI_fgx/HB_9slice_FineGlow.png.meta	2026-08-17 12:48:55.908493394 +0000
+++ b/Assets/UI_fgx/HB_9slice_FineGlow.png.meta	2026-08-17 12:48:55.918899003 +0000
@@ -49,7 +49,7 @@
   alignment: 0
   spritePivot: {x: 0.5, y: 0.5}
   spritePixelsToUnits: 100
-  spriteBorder: {x: 10, y: 10, z: 10, w: 10}
+  spriteBorder: {x: 5, y: 5, z: 5, w: 5}
   spriteGenerateFallbackPhysicsShape: 1
   alphaUsage: 1
   alphaIsTransparency: 1
